### PR TITLE
Play mode integration

### PR DESCRIPTION
**Pull request by miquelmiro3**
To test is add PLAY_BUILD to the preprocessor symbols

### CODE REVIEW

**Comment by miquelmiro3** [comment]
>

**Comment by miquelmiro3** [comment]
> Before merge with development it worked fine, but after the last commit this is what shows up on PLAY_BUILD image

Fixed with ur help ;)

File: Game/Assets/Scenes/first_deliver_scene.scene [hunk 1]
```diff
@@ -0,0 +1,1610 @@
+{
```
Comment (Bernatmago) [inline Game/Assets/Scenes/first_deliver_scene.scene:1]: What is this scene for?

**Review by Bernatmago — APPROVED** [review]
We will need some cleaning in the future but very good job!

**Review by VBoschMatas — APPROVED** [review]

**Pull request merged**